Source/ASDisplayNode+Layout.mm

```diff
@@ -143,7 +143,7 @@ - (NSString *)asciiArtName
 {
   NSString *string = NSStringFromClass([self class]);
   if (_debugName) {
-    string = [string stringByAppendingString:[NSString stringWithFormat:@"\"%@\"", _debugName]];
+    string = [string stringByAppendingString:[NSString stringWithFormat:@"\"%@\"",_debugName]];
```

Comment:
lol strange change. This will cause a spurious merge conflict with https://github.com/TextureGroup/Texture/pull/399 (which improves the perf) so better to revert it.